data.sql

```diff
@@ -1,4 +1,35 @@
-/* Populate database with sample data. */
+/* FEATURE BRANCH 03-querying-multiple-tables */
+
+-- D) INSERT values into the OWNERS table
+INSERT INTO owners (full_name, age) VALUES
+('Sam Smith', '34'),
+('Jennifer Orwell', '19'),
+('Bob', '45'),
+('Melody Pond', '77'),
+('Dean Winchester', '14'),
+('Jodie Whittaker', '38');
+
+-- E) INSERT values into the SPECIES table
+INSERT INTO species (name) VALUES
+('Pokemon'),
+('Digimon');
+
+-- F) INSERT Species IDs into the animals table
+UPDATE animals SET species_id = '2' WHERE name LIKE '%mon';
+UPDATE animals SET species_id = '1' WHERE species_id is NULL;
```

Comment:
- [ ] Please adhere to the project requirement, you should get the species ID using a query not `hard` coded the `id` of the species in order to update the data. Doing so will result in `inconsistency` and will not cover a few of the things. Just think about having a lot of species and it will be a `headache` for you to keep remembering the `ID` of the species.

### Requirement

![image](https://github.com/jcmunav63/vet_clinic_db/assets/66476329/c6bcebd7-b6e4-4c6f-8d38-14e1b86f3e88)